ci-automation/release.sh

```diff
@@ -119,6 +122,10 @@ function _inside_mantle() {
       done
     done
     for arch in amd64 arm64; do
+      if [[ "${arch}" == "arm64" ]] && [[ "${CHANNEL}" == "lts" ]]; then
```

Comment:
With the proposed fix above, this would not be needed.

Reply:
Wait, no. You'd need to change the loop in the line above to `for arch in "${arches[@]}"; do`.